meetings/2023-02-15.md

```diff
@@ -45,6 +51,28 @@
+* eP: We can discuss details now. HTTP Signatures is a valid direction. Some nuances: we currently work with Solid-OIDC with the assumption that the resource server shouldn't be too complex, and should have its own AuthN server. We leave it to implementation how to handle access tokens. We just specify the ID Token and ways to interact with AuthZ server to push id token in exchange for access token. The complexity can be on AuthZ server and resource server can focus on other things. With HTTP Signatures the Resource Server may need to  ... my main intention is to find a way to move forward and addressing those nuances with these use cases. Jackson proposed something, there are other approaches. In that work we should not separate AuthZ and AuthN. Maybe task force or dedicated meetings, they should be worked on together because they are hand in hand. 
+* SC: Henry raised that concern a while back. Not necessarily coupling, but thinking of the flow together. Let's keep this topic in the agenda for next week, compile a list of issues that touch both. The notification example: seems it's not a panel/notification specific problem, but something that we need to have a better view/design on the whole ecosystem. It's good to raise problem from strictly notification. 
+* eP: We have very clear use cases and implementations. CSS couples OIDC Provider (OP) and the Resource Server. I am using an authorization agent that needs to receive notifications so we have existing codebases with gaps that can give feedback to the spec work. 
+* SC: Do you think the PRotocol spec should be more clean on when to use Solid-OIDC? Because you mentioned it is social agent depends on a person. Whereas other cases (server-server), should we start mentioning these more clearly? Notifications as example: if notification server sends a message to a receiver, it is something we acknowledge for how some notification channels can be. Is that enough to raise it in PRotocol, Solid-OIDC for client-driven stuff, but if you want to do server-driven we don't have it nailed down yet but there is this other spec. Solid-OIDC or WebID-OIDC weren't the only way to authenticate, but only now we are trying to cover these major scenarios. 
+* eP: It'd be beneficial that spec clarifies Solid-OIDC is intended to be used by a user, possibly create an issue for server-side clients, and link to those as ongoing work. We should not only talk about authentication. We still miss similar AuthZ when using Solid-OIDC. We don't have a way for a user to authorize a client to access data. 
+* SC: Henry raised this in issues/discussions. The issued I linked (URL) there is one line: 
+  >Subscription Client lets the Notification Receiver know about the Notification Sender and their public key.
+  >Notification Receiver sets Authorization rules for Notification Sender.
+* SC: We need to allow receiver to add an authorization rule for who will have access to send a message. 
```

Comment:
Blockquotes mixed with lists are not handled well in Markdown. There's no clear way to end a blockquote except a blank line, which, within a list, has the undesired effect of putting a blank line between all list items.  So, I italicize the quote.
```suggestion
  _Subscription Client lets the Notification Receiver know about the Notification Sender and their public key._
  _Notification Receiver sets Authorization rules for Notification Sender._
* SC: We need to allow receiver to add an authorization rule for who will have access to send a message. 
```